canonical-web-and-design/maas-ui
# feat(ui): remove RSD components from UI

## Done

- Removed RSD components from UI since it will no longer be supported from 2.10

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- With a MAAS that has RSD pods (`make sampledata` creates some), check that there is longer a navigation item for RSD in the angular or react apps
- Check that composing machines from a virsh/lxd pod still works properly
- Go to the machine list and click "Add hardware"
- Check that there is no option for RSD

## Fixes

Fixes #2097
